index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/8f283d01ad.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>Daron Wiafe (Software Developer)</title>
+</head>
+<body>
+    <div id="wrapper">
+        <header class="intro">
+            <nav class="portfolio_nav">
+                <a href="#" class="home_link">Daron Wiafe</a>
+                <a href="#" class="menu_dropdown"><i class="fas fa-bars fa-2x"></i></a>
+                <ul class="links">
+                    <li><a>Portfolio</a></li>
+                    <li><a>About</a></li>
+                    <li><a>Contact</a></li>
+                </ul>
+            </nav>
```

Comment:
- [ ] Your `<header>` must be the part of your portfolio that contains the logo and menus. Please fix the range of your `<header>` according to the image below. You will create a common adopted user interface and a cleaner code.
<img width="594" alt="Screen Shot 2022-08-18 at 7 51 29 AM" src="https://user-images.githubusercontent.com/47673311/185378702-519f7c9c-b8b5-4645-8cbb-7d469afe6d24.png">

- [ ] The `topcircle.png` need to be styled as the [background image](https://www.w3schools.com/html/html_images_background.asp) in the CSS. Kindly remove the `<img>` tag from your HTML and define the image in your CSS.

- [ ] The `<h1>` and the `<h2>` tags must be placed inside the next section.